torch/csrc/jit/script/init.cpp

```diff
@@ -348,9 +345,12 @@ void initJitScriptBindings(PyObject* module) {
         return bool(self.find_method(name));
       })
       .def("_method_names", [](Module& self) {
-        return fmap(self.get_methods(), [](const std::unique_ptr<Method> & m) {
-          return m->name();
-        });
+        std::vector<std::string> method_names;
+        method_names.reserve(self.get_methods().size());
+        for (auto& method : self.get_methods()) {
+          method_names.push_back((*method)->name());
+        }
+        return method_names;
```

Comment:
What's wrong with `fmap`?

Reply:
Can do. When we replace the script module with the new module, we'll be able to do `module.methods().apply()` hopefully